### ISSUE TITLE
Title metadata should use track title instead of recording title

### ISSUE THREAD

**Issue by jasLogic** [cyanreg/cyanrip#56]
At the moment the title is read from the MusicBrainz **record** title if present. I think it would make more sense to use the **track** title in every case instead.

The reason for this is that MusicBrainz allows one record to have multiple tracks in different albums and the track title of the album might be different than the recording title. So in my opinion it would make sense to have the **track** title as listed on the CD / booklet instead of the **recording** title which might be different.

(Thanks for the great software btw, it works really well :) )

**Comment by cyanreg** [comment]
Post an example?

**Comment by jasLogic** [comment]
Sure, I noticed it [here](https://musicbrainz.org/release/88e1cd32-9c34-40e2-a4d0-af3af3144ad6) where most track titles are quite different from the recording titles.

But there are also some other examples like live recordings (for examples [this](https://musicbrainz.org/recording/3a4fa843-ccfe-4f1d-8bd7-6f718bc5cbe7) one) where the recording title does not have "(live)" in the title but the track title of the album does.

And I think that the title in the metadata (and the title of the file) should match the track title of the album rather than the recording title.

**Comment by cyanreg** [comment]
So instead of "Three Parts on a Ground in D, Z. 731" you're getting "Fantasia, 3 parts on a Ground for three recorders, Z. 731"?

**Comment by jasLogic** [comment]
No, I'm getting "Three Parts upon a Ground in D major, Z 731" which is the title of the [recording](https://musicbrainz.org/recording/d561815e-4ccb-4cd6-8e4d-1ec455e50b1b). You can view the recording by clicking on the title of the track.

I think "Fantasia, 3 parts on a Ground for three recorders, Z. 731" is the corresponding [work](https://musicbrainz.org/doc/Work) of the recording. So one work can have different recordings and one recording can have different tracks / releases.

**Comment by cyanreg** [comment]
So which one would you like? is "Three Parts upon a Ground in D major, Z 731" not what is printed on the CD?
Is the MB entry wrong?

**Comment by jasLogic** [comment]
Sorry for the confusion,

the **track** title is "Three Parts on a Ground in D, Z. 731",
the **recording** title is "Three Parts upon a Ground in D major, Z 731" and
the **work** title is "Fantasia, 3 parts on a Ground for three recorders, Z. 731".

The **track** title is the name of the song as given in the album and on the CD. I checked that the MusicBrainz entry for this is the correct title as written on the CD. This **track** title is what I think should be the title which cyanrip uses for the file name and title metadata.

The **recording** title is what is currently used (if available) but which may not represent the name of the song on the CD (which is the case here as it is slightly different).

The **work** title is something different and not relevant here.

**Comment by meersjo** [comment]
Could this not be solved the simple way by adding formatting tags for the various titles in MB?
